<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Click to see Screenshot]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Wed_Aug_03_13_41_15_IST_2022.jpg>Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Wed_Aug_03_13_41_15_IST_2022.jpg><img src=Wed_Aug_03_13_41_15_IST_2022.jpg height=200 width=200></img></a>]]>
    </line>
  </reporter-output>
  <suite started-at="2022-08-03T13:39:26 IST" name="Page Object Model" finished-at="2022-08-03T13:41:15 IST" duration-ms="109055">
    <groups>
    </groups>
    <test started-at="2022-08-03T13:39:26 IST" name="Login Test" finished-at="2022-08-03T13:39:30 IST" duration-ms="3489">
      <class name="testcases.LoginTest">
        <test-method signature="loginTest(java.util.Hashtable)[pri:0, instance:testcases.LoginTest@67c119b7]" started-at="2022-08-03T13:39:26 IST" name="loginTest" data-provider="dp" finished-at="2022-08-03T13:39:29 IST" duration-ms="3251" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Tony@7799, runmode=Y, username=itoney7799@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- testcases.LoginTest -->
    </test> <!-- Login Test -->
    <test started-at="2022-08-03T13:39:30 IST" name="Create Account Test" finished-at="2022-08-03T13:41:15 IST" duration-ms="105544">
      <class name="testcases.CreateAccountTest">
        <test-method signature="createAccountTest(java.util.Hashtable)[pri:0, instance:testcases.CreateAccountTest@1fcf9739]" started-at="2022-08-03T13:39:30 IST" name="createAccountTest" data-provider="dp" finished-at="2022-08-03T13:41:15 IST" duration-ms="105102" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{accountname=Raman}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Create account test failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Create account test failed
at org.testng.Assert.fail(Assert.java:99)
at testcases.CreateAccountTest.createAccountTest(CreateAccountTest.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Click to see Screenshot]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Wed_Aug_03_13_41_15_IST_2022.jpg>Screenshot</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Wed_Aug_03_13_41_15_IST_2022.jpg><img src=Wed_Aug_03_13_41_15_IST_2022.jpg height=200 width=200></img></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- createAccountTest -->
      </class> <!-- testcases.CreateAccountTest -->
      <class name="testcases.LoginTest">
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testcases.LoginTest@67c119b7]" started-at="2022-08-03T13:41:15 IST" name="tearDown" finished-at="2022-08-03T13:41:15 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.LoginTest -->
    </test> <!-- Create Account Test -->
  </suite> <!-- Page Object Model -->
</testng-results>
